crates/libtiny_tui/src/tui.rs

```diff
@@ -1058,7 +1130,9 @@ impl TUI {
                     .splice(insert_idx..insert_idx, to_move.into_iter());
                 self.select_tab(insert_idx);
             }
-        } else if !self.is_server_tab(self.active_idx - 1) {
+        }
+        // Misc tabs get captured in server group
```

Comment:
Hm.. this is a bit unexpected, but I guess there's no other way to implement it, so it's fine.

Reply:
I thought you wanted this 😂